reference/yaml-format-reference.md

```diff
@@ -1,88 +1,117 @@
----
-description: Declaring and configuring Recipes and Styles in YAML
----
-
 # Declarative YAML format
 
-There are two places OpenRewrite [YAML](https://yaml.org) files may appear:
+OpenRewrite allows you to create [recipes](/concepts-and-explanations/recipes.md) and [styles](/concepts-and-explanations/styles.md) in [YAML](https://yaml.org). While doing so potentially reduces customizability, it makes up for that with development speed and portability.
 
-* Within the rewrite.yml file of a project that applies rewrite recipes via the [rewrite-gradle-plugin](gradle-plugin-configuration.md) or [rewrite-maven-plugin](rewrite-maven-plugin.md)
-  * Will not be included in jars published from your project
-* Inside the META-INF/rewrite folder of a jar
-  * Put your recipes here if you want to redistribute and apply them to other repositories
-  * See [Recipe Development Environment](../authoring-recipes/recipe-development-environment.md) for instructions on setting up your project for recipe development
-  * e.g.: [rewrite-testing-frameworks](https://github.com/openrewrite/rewrite-testing-frameworks/tree/master/src/main/resources/META-INF/rewrite)
+To help you confidently define recipes and styles in YAML, this guide will walk you through all of the ways you can configure an OpenRewrite YAML file.
 
-In either case, these YAML files share the same format and purpose: making styles and recipes available to rewrite. A single OpenRewrite YAML file may contain any number of recipes and styles, separated by `---`.
+## Where OpenRewrite YAML files can exist
 
-{% hint style="info" %}
-Within a YAML document recipe and style names are always fully qualified.
-{% endhint %}
+There are two places where you can define an OpenRewrite YAML file:
 
-{% hint style="warning" %}
-Don't place your custom recipes into the org.openrewrite namespace. We recommend using the same "reverse domain name" naming scheme used in Java packages.
-{% endhint %}
+1. Within the `rewrite.yml` file of a project that applies rewrite recipes via the [rewrite-gradle-plugin](gradle-plugin-configuration.md) or [rewrite-maven-plugin](rewrite-maven-plugin.md)
+2. Inside the `META-INF/rewrite` folder of a jar (such as in the [rewrite-testing-frameworks](https://github.com/openrewrite/rewrite-testing-frameworks/tree/master/src/main/resources/META-INF/rewrite))
+
+If you define a recipe or style in the `rewrite.yml` file, they _will not_ be included in the jars published from your project.
+
+If you want to redistribute a recipe or a style and apply them to other projects, you'll need to create them inside of the `META-INF/rewrite` folder of a jar.
```

Comment:
```suggestion
If you want to distribute a recipe or a style and apply them to other projects, you'll need to create them inside of the `META-INF/rewrite` folder of a jar.
```
Might be me; but the `re` seemed off. But who knows, it's not my first language.

Reply:
Also seen once more below. Only pointed out here.

Reply:
Thanks for the call-out. I grabbed that line from the existing doc, but I think I agree with you so I'll update those places.